src/vmm/src/config/mod.rs

```diff
@@ -119,10 +144,20 @@ pub struct KernelConfig {
     pub himem_start: u64,
 }
 
-impl TryFrom<String> for KernelConfig {
+impl Default for KernelConfig {
```

Comment:
I would prefer to remove this so that people do not use by mistake. `KernelConfig` should not support a default.